addons/im_livechat/static/tests/embed/feedback_panel.test.js

```diff
@@ -142,3 +143,34 @@ test("Start new session from feedback panel", async () => {
     await contains(".o-mail-ChatWindow", { count: 1 });
     await contains(".o-mail-ChatWindow", { text: "Bob Operator" });
 });
+
+test("open review link on good rating", async () => {
+    patchWithCleanup(window, {
+        open: (...args) => {
+            expect.step("window.open");
+            expect(args[0]).toBe("https://www.odoo.com");
+            expect(args[1]).toBe("_blank");
+        },
+    });
+    await startServer();
+    await loadDefaultEmbedConfig();
+    const operatorUserId = serverState.userId;
+    await start({ authenticateAs: false });
+    await click(".o-livechat-LivechatButton");
+    await contains(".o-mail-ChatWindow");
+    await insertText(".o-mail-Composer-input", "Hello World!");
+    triggerHotkey("Enter");
+    await contains(".o-mail-Message-content", { text: "Hello World!" });
+    // simulate operator leaving
+    await withUser(operatorUserId, () =>
+        getService("orm").call("discuss.channel", "action_unfollow", [
+            [Object.values(getService("mail.store").Thread.records).at(-1).id],
+        ])
+    );
```

Comment:
Whats's the need for operator to leave? Guest can close the livechat himself too.